### ISSUE TITLE
Font style behavior

### ISSUE THREAD

**Issue by BrainMaestro** [sharkdp/bat#130]
![screen shot 2018-05-18 at 2 24 52 pm](https://user-images.githubusercontent.com/1738373/40234520-73b8d314-5aa7-11e8-8efc-9e1648146a3f.png)

Is that expected behavior? I believe it's from the font styles (probably bold). This is with iTerm2 on macOS.

**Comment by sharkdp** [comment]
Yes, this is probably due to the changes in #96. It works fine for me on Linux/{terminator,xterm,urxvt}: these keywords should be in *italic* font style.

Not sure if this is related: https://alexpearce.me/2014/05/italics-in-iterm2-vim-tmux/

**Comment by sharkdp** [comment]
Do you know if italic text will work with a standard iTerm2 or will this be an issue for every MacOS user?

**Comment by BrainMaestro** [comment]
I don't think it will
